util/test/sub_clean

```diff
@@ -11,6 +11,8 @@ def getExecname(f):
         return f[:len(f)-5]
     elif ((f.count('.') > 1) and (len(f) >= 8) and (len(f)-f.find('.test.c')==7)):
         return f[:len(f)-7]
+    elif ((f.count('.') > 1) and (len(f) >= 11) and (len(f)-f.find('.ml-test.c')==10)):
+        return f[:len(f)-10]
```

Comment:
I know you're just following what was here, but this is mortifying :)

Maybe something more like:

```python
# return execname if f has a testable extension, None otherwise
def getExecname(f):
    if f.endswith((".chpl", ".test.c", ".ml-test.c")):
        return os.path.splitext(f)[0]
    return None
```
?